migrations/versions/940_brief_response_award_details.py

```diff
@@ -0,0 +1,34 @@
+"""award_details JSON blob and awarded_at date stamp on BriefResponse
+Constraint on Brief to allow only one BriefResponse with non-null 'awarded_at'
+
+Revision ID: 940
+Revises: 930
+Create Date: 2017-08-07 15:22:43.619680
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '940'
+down_revision = '930'
+
+
+def upgrade():
+    op.add_column('brief_responses', sa.Column('award_details', postgresql.JSON(astext_type=sa.Text()), nullable=True))
```

Comment:
Worth defaulting to '{}' here and removing checks like `br.award_status and br.award_status.get('foo') == 'bar'`
To do this you can add a `server_default` to the column declaration to set a default in the database itself.